packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -420,6 +420,18 @@ describe('I18n', () => {
       expect(i18n.formatDate(date, {timeZone: timezone})).toBe(expected);
     });
 
+    it('formats a date using DateStyle.ExtraLong', () => {
+      const date = new Date('2012-12-20T00:00:00-00:00');
+      const i18n = new I18n(defaultTranslations, {
+        ...defaultDetails,
+        timezone,
+      });
+
+      expect(i18n.formatDate(date, {style: DateStyle.ExtraLong})).toBe(
+        'December 20, 2012, 11:00 AM',
```

Comment:
Shouldn't you have the weekday here? Like `Thursday, December 20, 2012 00:00` that you have in the doc?

Reply:
The doc example is a mistake. The weekday should not be included.

Reply:
Oh ok then! 👍